roles/openshift_hosted/tasks/router.yml

```diff
@@ -61,25 +61,24 @@
 - name: Create the router service account(s)
   oc_serviceaccount:
     name: "{{ item.serviceaccount }}"
-    namespace: "{{ item.namespace }}"
+    namespace: "{{ item.namespace | default('default') }}"
```

Comment:
Namespace is a required field in `openshift_hosted_routers`, there is no need to be defaulted